Source/Extensions/UIViewExtensions.swift

```diff
@@ -68,3 +68,72 @@ extension UIView {
   }
 
 }
+
+extension UIView {
+
+    func addBorder(with list: [SideBorder], thickness: Double, color: UIColor) {
+        list.forEach {
+            addBorder(side: $0, thickness: thickness, color: color)
+        }
+    }
+
+    private func addBorder(side: SideBorder, thickness: Double, color: UIColor) {
+        var thickness = thickness
+        var edgeView = UIView().disabledAutoresizingIntoConstraints()
+                edgeView.backgroundColor = color
+
+        switch side {
+            case .left(let edge):
+                if let edge = edge {
+                    edgeView = edge
+                    thickness = edgeView.frame.width
+                }
+                addSubview(edgeView)
+                NSLayoutConstraint.activate([
+                    edgeView.topAnchor.constraint(equalTo: topAnchor),
+                    edgeView.leftAnchor.constraint(equalTo: leftAnchor),
+                    edgeView.bottomAnchor.constraint(equalTo: bottomAnchor),
+                    edgeView.widthAnchor.constraint(equalToConstant: thickness)
```

Comment:
Just curious, but why left anchor and width, instead of left anchor and right anchor?